plugins/board-resources/src/utils/ClickOutside.ts

```diff
@@ -0,0 +1,16 @@
+/** Dispatch event on click outside of node */
+export function clickOutside(node: any) {
+  const handleClick = (event: any) => {
+    if (node && !node.contains(event.target) && !event.defaultPrevented) {
+      node.dispatchEvent(new CustomEvent('click_outside', node))
+    }
+  }
+
+  document.addEventListener('click', handleClick, true)
```

Comment:
I suppose you could use <svelte:window on:click={}>

Reply:
Yes, switched to it. The only problem I see is that now window callbacks live until the component is destroyed, whereas when i used `use:` derective with `document.addEventListener` they were removed when `Textarea` were closed.